src/test/java/uk/gov/pay/directdebit/mandate/services/gocardless/GoCardlessMandateStateUpdaterTest.java

```diff
@@ -21,11 +22,11 @@
 @RunWith(MockitoJUnitRunner.class)
 public class GoCardlessMandateStateUpdaterTest {
 
-    private static final GoCardlessMandateId GOCARDLESS_MANDATE_ID = GoCardlessMandateId.valueOf("MD123");
-    private static final GoCardlessOrganisationId GOCARDLESS_ORGANISATION_ID = GoCardlessOrganisationId.valueOf("OR123");
+    private static final GoCardlessMandateIdAndOrganisationId GOCARDLESS_MANDATE_ID_AND_ORGANISATION_ID = new GoCardlessMandateIdAndOrganisationId(
```

Comment:
might be a nice use of `var` since its clear what is being assigned.

Reply:
Unfortunately, we can’t use `var` here because it’s a field, not a local variable